tests/test_input_model.py

```
@@ -920,6 +920,54 @@ def test_input_model_custom_generic_type_import(tmp_path: Path) -> None:
+
+    import ast
+    from typing import Literal, get_args, get_origin
+
+    from tests.data.python.input_model.literal_enum_first import Status as FirstStatus
+    from tests.data.python.input_model.literal_enum_second import Status as SecondStatus
+    from tests.data.python.input_model.pydantic_models import LiteralFlagValue, LiteralIntFlagValue
+
+    tree = ast.parse(output_path.read_text(encoding="utf-8"))
+    namespace: dict[str, object] = {}
+    import_nodes = tuple(node for node in tree.body if isinstance(node, (ast.Import, ast.ImportFrom)))
+    exec(compile(ast.Module(body=list(import_nodes), type_ignores=[]), str(output_path), "exec"), namespace)
```

Ruff (0.16.0)

[error] Use of `exec` detected

(S102)